tests/robot-tests/tests/libs/common.robot

```diff
@@ -1,7 +1,7 @@
 *** Settings ***
-Library     SeleniumLibrary    timeout=${timeout}    implicit_wait=${implicit_wait}    run_on_failure=do this on failure
-Library     OperatingSystem
 Library     Collections
+Library     OperatingSystem
+Library     SeleniumLibrary    timeout=${timeout}    implicit_wait=${implicit_wait}    run_on_failure=do this on failure
```

Comment:
It looks like the formatter's reordering all of the libraries incorrectly. I specifically changed the configuration to avoid it doing stuff like this as it can break the Robot tests.

How did you run the formatter? You should ideally use the `format:robot` command in the root `package.json` to avoid running the wrong command.

Would revert all of the formatter changes and re-run it with the proper command.

Reply:
I usually run `robotidy --config tests/robot-tests/robotidy.toml tests/robot-tests` as there is an issue at the moment when lint-staged runs the format command (`C:\Users\hive\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\python.exe: No module named pipenv`)